docs/UserGuide.md

```diff
@@ -69,148 +142,166 @@ Shows a message explaining how to access the help page.
 
-Adds an order to an existing person in the address book.
+🌟 Keep a comprehensive record of your customers' purchases by using addorder. This feature allows you to track sales trends, member preferences, and reward loyalty, enhancing customer satisfaction and retention.
 
 Format: `addorder n/MEMBER_NAME o/ORDER_DETAILS`
 
-* The order will be added to the person with the closest resembling name to `MEMBER_NAME`.
-* If no person with a resembling name is found, no order will be added to any person.
-* All persons are considered, not just those in the displayed person list.
+* The order will be added to the member with the closest resembling name to `MEMBER_NAME`.
+* If no member with a resembling name is found, no order will be added to any member.
+* All members are considered, not just those in the displayed member list.
 * `ORDER_DETAILS` must not be empty.
 
+💡 Use unique attributes of members when adding points to avoid errors. Partial names are matched to the closest resembling member.
+
 Examples:
 * `addorder n/John Doe o/Butter Cake` Adds an order of `Butter Cake` to `John Doe`
 * `addorder n/Betsy o/200g Macadamia Nut Cookies` Adds of order of `200g Macadamia Nut Cookies` to `Besty Crower`
 
-### Deleting orders of a person: `delorder`
+### Adding points: `addpoints`
+
+Adds points for a member in the loyalty program.
 
-Adds an order to an existing person in the address book.
+🌟 Encourage repeat business and reward customer loyalty by using addpoints to add reward points to members' accounts. This helps keep your customers engaged and appreciative of the value they get from shopping at your bakery.
 
-Format: `delorder n/MEMBER_NAME i/ORDER_INDEX`
+Format: `addpoints n/MEMBER_NAME p/POINTS`
 
-* The order with corresponding `ORDER_INDEX` will be removed from the person with the closest resembling name to `MEMBER_NAME`.
-* If no person with a resembling name is found, no order will be removed from any person.
-* All persons are considered, not just those in the displayed person list.
-* `ORDER_INDEX` **must be a positive integer** 1, 2, 3, ...
+💡 Use unique attributes of members when adding orders to avoid errors. Partial names are matched to the closest resembling member.
+
+* Adds the corresponding number of points for a member based on their name to the current points they have.
+* The points **must be a positive integer** 1, 2, 3, … 200
 
 Examples:
-* `delorder n/John Doe i/1` deletes the first order of `John Doe`
+* `addpoints n/John Doe p/50`
 
+### Updating membership: `addmship`
 
-### Editing a person : `edit`
+Updates the membership tier of a member in the loyalty program.
 
-Edits an existing person in the address book.
+🌟 Motivate your customers to keep coming back by using addmship to upgrade their membership tier. Higher tiers can offer better rewards, creating a sense of achievement and encouraging more purchases.
+
+Format: `addmship n/MEMBER_NAME ms/MEMBERSHIP_TEIR`
```

Comment:
just a small spelling mistake should be "tier"

Reply:
Thank you for pointing it out!